core/plannernotes.c

```diff
@@ -265,28 +293,54 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					len += snprintf(buffer + len, sz_buffer - len, "<td style='padding-left: 10px; float: right;'>%s</td>", temp);
 				}
 
+				if (istrimix) {
+					runtime = (dp->time + 30) / 60;
+					deltaN2 = ((1000 - newgasmix.he.permille - newgasmix.o2.permille) - (1000 - lastprintgasmix.he.permille - lastprintgasmix.o2.permille)) / 10.0;
```

Comment:
Can we put these calculations in a helper function that we can then reuse to display an ICD event in the profile?